eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -33,7 +33,7 @@ def start(queue, options = {}, &block)
         configure(queue, options)
 
         if options[:client] == nil
-          raise "[#{self.class}] - :client (QueueClient) must be specified."
+          EventQ.log(:info, "[#{self.class}] - options[:client] is now deprecated!!, please pass options[:mq_endpoint].")
```

Comment:
the comment here in the log message is about `mq_endpoint` which is for rabbitmq and not aws.  But this is the aws worker.